homestead.md

```diff
@@ -129,13 +127,15 @@ Homestead runs on any Windows, Mac, or Linux system, and includes Nginx, PHP, My
 <a name="first-steps"></a>
 ### First Steps
 
+> {note} If you are using Windows 10 with Hyper-V, ideally you can use VirtualBox 6.x or recent, as it will use Hyper-V as hypervizor and allow you to continue to use Hyper-V for what you needed for without affecting anything. Vagrant with VirtualBox on top of Hyper-V, solves the private host only networking issues that Vagrant can't handle directly with Hyper-V.
```

Comment:
I've read about this, but haven't tried it. Wouldn't want to recommend this until I've been able to verify this doesn't have unintended consequences.

Reply:
Could you detail this? Please test, I am using it like this since Virtualbox got Hyper-V support. It's best way for using Vagrant in Windows, as Vagrant has poor support for Hyper-V networking (possible will be fixed in a few months).